fixtures/light4me/bros-5-beam.json

```diff
@@ -0,0 +1,1488 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "BROS 5 BEAM",
+  "shortName": "b5b",
+  "categories": [
+    "Moving Head",
+    "Pixel Bar",
+    "Strobe",
+    "Color Changer"
+  ],
+  "meta": {
+    "authors": [
+      "Rainer Jung"
+    ],
+    "createDate": "2025-05-20",
+    "lastModifyDate": "2025-05-23"
+  },
+  "links": {
```

Comment:
Please could you add a link to the manual?

Reply:
I'd love to, but I didn't find any website of light4me, there are links to manualslib or similar, but not original link.

Reply:
I checked again, there's https://light4me.pl/en/, but they don't show the product any more and so far I didn't find any manual there as well.

Reply:
I could add one of these, but they are not from the producer:
- https://www.musicexpress.pl/amfile/file/download/file/248/product/12720/
- https://www.manualslib.de/manual/1313181/Light4Me-Bros-5-Strahl.html